src/utils/iosFix.js

```diff
@@ -11,6 +11,17 @@ export const iOSfix = () => {
     document.body.style.top = `${offset * -1}px`
     dom.addClass(document.body, swalClasses.iosfix)
     lockBodyScroll()
+    addBottomPaddingForTallPopups() // #1948
+  }
+}
+
+const addBottomPaddingForTallPopups = () => {
+  const safari = !navigator.userAgent.match(/CriOS/i)
+  if (safari) {
+    const bottomPanelHeight = 44
+    if (dom.getPopup().scrollHeight > window.innerHeight - bottomPanelHeight) {
+      dom.getContainer().style.paddingBottom = `${bottomPanelHeight}px`
```

Comment:
How about adding a css class instead of using a fixed value for the padding bottom? I mean, there could be a class like `.ios-popup-padding-bottom` using the `safe-area-inset-bottom` value leaving for the iOS to define this value so if it changes or varies depending on the device, you don't need to fix it and put another `if` in the code.

Reply:
It seems that `safe-area-inset-bottom` isn't defined for iPhone 7, that's why I wasn't able to reproduce your PR on my device. The issue though does exist on iPhone 7.

Also csstree parser can't handle `env(safe-area-inset-bottom)` for some reason. 

So let's use this simpler and more obvious way instead of `safe-area-inset-bottom`.

Reply:
@tiagostutz can you verify on your side that the tall popup example on https://deploy-preview-1993--sweetalert2.netlify.app/ works?

Reply:
> It seems that `safe-area-inset-bottom` isn't defined for iPhone 7, that's why I wasn't able to reproduce your PR on my device. The issue though does exist on iPhone 7.
> 
> Also csstree parser can't handle `env(safe-area-inset-bottom)` for some reason.
> 
> So let's use this simpler and more obvious way instead of `safe-area-inset-bottom`.

Ok!



> @tiagostutz can you verify on your side that the tall popup example on https://deploy-preview-1993--sweetalert2.netlify.app/ works?

Tested here in Safari with Simulator of iPhone X, a real iPhone XR and real iPhone 8 and all worked!